lib/rspec/matchers/built_in/eq.rb

```diff
@@ -20,7 +20,8 @@ def failure_message_when_negated
         # @api private
         # @return [String]
         def description
-          "#{name_to_sentence} #{@expected.inspect}"
+          english_name = EnglishPhrases.humanize(name)
+          "%s %s" % [english_name, @expected.inspect]
```

Comment:
OK, I'll switch back to interpolation.  If performance were not a concern, I like format strings because the non-interpolated content (in this case, whitespace) is easier to see.